app/src/controllers/filter/selectors.js

```diff
@@ -68,3 +72,17 @@ export const dirtyFilterIdsSelector = createSelector(
       })
       .map((item) => item.id),
 );
+
+export const filterFromQuerySelector = createSelector(
+  queryParametersSelector,
+  ({ [SIZE_KEY]: size, [SORTING_KEY]: sort, [PAGE_KEY]: page, ...filter }) => {
+    if (isEmptyObject(filter)) {
+      return null;
+    }
+
+    const [[filterName, filterValue]] = Object.entries(filter);
```

Comment:
Actually there can be several filters (even if now we don't support such case from email notifications, it can be useful in the future f.e. by creating a link to a group of launches from the client/agent side).
Therefore, please parse all possible filters accordingly, not just only one.